pkg/osquery/tables/execparsers/softwareupdate/parse_test.go

```diff
@@ -0,0 +1,152 @@
+//go:build darwin
+// +build darwin
+
+package softwareupdate
+
+import (
+	"bytes"
+	_ "embed"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed test-data/beta-update-available-noscan.txt
+var beta_update_available_noscan []byte
```

Comment:
Probably not worth changing, but given the number of these I'd probably _not_ embed, and instead make something like `readTestFile(t, filename) []byte` and stick that into the table definition